anitya/api_v2.py

```diff
@@ -18,19 +18,100 @@
 _BASE_ARG_PARSER.add_argument('access_token', type=str)
 
 
+def _page_validator(arg):
+    """Validator for a pagination page number."""
+    arg = int(arg)
```

Comment:
No, the flask-restful middleware will catch the exception and turn it into a pretty HTTP 400 with details about what went wrong. I did neglect to add tests for that though (which is why codecov was displeased with me). You can check out the tests to see what happens when the cast to int fails :)